common/on_actions/01_pop_on_actions.txt

```diff
@@ -0,0 +1,13 @@
+#When adding effects directly in on_actions always consider if it would be best to do so through a scripted effect. on_actions with a lot of script in them are harder to get an overview of.
+
+# country random events
+on_yearly_pulse = {
+	events = {
+		migrations.101
+		migrations.102
+		population_grow.101
+		population_grow.104
+		population_upgrade.101
+		population_upgrade.104
```

Comment:
In on_yearly_pulse on_actions, doesn't the game choose one of the events at random?

If that's the case, we can create a single umbrella event that calls those 6.

Reply:
no. that is for random_events. if it is in events it calls all of them